client/my-sites/marketing/connections/connection.jsx

```diff
@@ -250,15 +239,37 @@ class SharingConnection extends Component {
 	}
 }
 
+const SharingConnectionKeyringUserLabel = localize(
+	( { siteId, keyringUserId, translate, userId } ) => {
+		const fetchOptions = {
+			search: keyringUserId,
+			search_columns: [ 'ID' ],
+		};
+
+		const { data } = useUsersQuery( siteId, fetchOptions );
```

Comment:
This is undefined for the editor role and below:

<img width="1059" alt="Screen Shot 2021-04-08 at 2 31 36 PM" src="https://user-images.githubusercontent.com/1270189/114099098-25583500-9877-11eb-909b-fae84afce711.png">